README.md

```diff
@@ -86,16 +86,56 @@ directory (in this case the `$HOME/google-drive`):
 First install the `inotify-tools` (seems to be named like that in all major distros): 
 test that it works by calling `inotifywait -h`.
 
-Prepare a Google Drive folder in your $HOME directory with `grive -a`.
+Ensure you have first initialized a Google Drive folder per the instructions in the usage section.
+Execute these commands from the directory above the folder you are using. In the example below,
+`google-drive` was initialized inside the user `$HOME` directory.
 
 ```bash
-# 'google-drive' is the name of your Google Drive folder in your $HOME directory
 systemctl --user enable grive-timer@$(systemd-escape google-drive).timer
 systemctl --user start grive-timer@$(systemd-escape google-drive).timer
 systemctl --user enable grive-changes@$(systemd-escape google-drive).service
 systemctl --user start grive-changes@$(systemd-escape google-drive).service
 ```
 
+For example:
+```
```

Comment:
Isn't there a nice tutorial this can link to? It's basically explaining how `systemctl --user` works under the hood and has nothing to do with grive itself.